reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1297 @@ title: Style
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [View styles](#view-styles-props)
+- [Text styles](#text-styles-props)
+- [Image styles](#image-styles-props)
+
+⚠️ _Internally, React Native codebase types for this function are called
+`____DangerouslyImpreciseStyle_Internal`. That's why we created new functions:
+[`viewStyle`](#style-viewstyle), [`textStyle`](#style-textstyle) &
+[`imageStyle`](#style-imagestyle)_.
+
+### `Style.viewStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [View styles](#view-styles-props)
+
+### `Style.textStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shadow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [Text styles](#text-styles-props)
+
+### `Style.imageStyle`
+
+This function accept all React Native following styles
+
+- [Layout styles](#layout-styles-props)
+- [Shadow styles](#shad@ow-styles-props)
+- [Transform styles](#transform-styles-props)
+- [Image styles](#image-styles-props)
+
+## Style Props
+
+### Layout Style Props
+
+[Official documentation](https://facebook.github.io/react-native/docs/layout-props)
+
+#### `alignContent`
+
+Accept one of the following values:
+
+- `` `flexStart `` (default)
+- `` `flexEnd ``
+- `` `center ``
+- `` `stretch ``
+- `` `spaceAround ``
+- `` `spaceBetween ``
+
+Controls how rows align in the cross direction, overriding the `alignContent` of
+the parent.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content)
+- [Yoga reference](https://yogalayout.com/docs/align-content/)
+
+#### `alignItems`
+
+Accept one of the following values:
+
+- `` `flexStart ``
+- `` `flexEnd ``
+- `` `center ``
+- `` `stretch `` (default)
+- `` `baseline ``
+
+Aligns children in the cross direction. For example, if children are flowing
+vertically, `alignItems` controls how they align horizontally.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)
+- [Yoga reference](https://yogalayout.com/docs/align-items/)
+
+#### `alignSelf`
+
+Accepts one of the following values:
+
+- `` `auto `` (default)
+- `` `flexStart ``
+- `` `flexEnd ``
+- `` `center ``
+- `` `stretch ``
+- `` `baseline ``
+
+Controls how a child aligns in the cross direction, overriding the `alignItems`
+of the parent.
+
+- [Web reference](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self)
+- [Yoga reference](https://yogalayout.com/docs/align-items/)
+
+#### `aspectRatio`
+
+Accepts a `float`.
+
+Aspect ratio control the size of the undefined dimension of a node. _Aspect
+ratio is a non-standard property only available in react native and not CSS._
+
+- On a node with a set width/height aspect ratio control the size of the unset
```

Comment:
adding a comma and fixing subject-verb agreement:
width/height, aspect ratio controls